rero_mef/cli.py

```
@@ -1091,25 +1099,57 @@ def queue_count():
+        count = 0
+        with current_celery.connection_or_acquire() as connection:
+            channel = connection.channel()
+            try:
+                for queue_name in queues:
+                    try:
+                        _, message_count, _ = channel.queue_declare(
+                            queue=queue_name, passive=True
+                        )
+                        count += message_count
+                    except Exception:
+                        continue
```

Ruff (0.15.7)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)